Userland/Applications/Spreadsheet/Spreadsheet.h

```diff
@@ -22,6 +22,20 @@
 
 namespace Spreadsheet {
 
+class CellChange : public RefCounted<CellChange> {
```

Comment:
This doesn't seem to be benefiting from being refcounted, is there any particular reason that it is?